src/run_cmd.js

```diff
@@ -37,7 +37,9 @@ export default function runCmd (cwd, argv) {
   }, function (err, statuses) {
     if (err) throw err
     const info = scripts.map((script, i) =>
-      script + ' exited with status ' + statuses[i]
+      pkg.name + ': '
+      + script + ' exited with status ' + statuses[i] + ' '
+      + 'in directory ' + cwd
```

Comment:
I was assuming this file was also used when running lifecycle scripts of dependencies. Is that not the case?

And is package.json always the root package.json then?